src/main/groovy/de/triplet/gradle/play/PlayPublishListingTask.groovy

```diff
@@ -40,6 +40,10 @@ class PlayPublishListingTask extends PlayPublishTask {
     publishListing() {
         super.publish()
 
+        if (!inputFolder.exists()) {
```

Comment:
We can move this code above the `super.publish()` call to avoid executing even more code